rom-hacking/scripting/animation-commands/ac-world-x.md

```diff
@@ -0,0 +1,34 @@
+# (786) _AC_WORLD_X
+
+## Effect
+
+Moves the actor of the animation to a tile on the x-axis of the current map over a given amount of frames per tile.
```

Comment:
Might be worth mentioning in the info blurb (with a picture maybe?) that X is horizontal and Z is vertical, with (0, 0) being top left.